src/watertap_contrib/seto/unit_models/surrogate/tests/test_chemical_softening.py

```diff
@@ -0,0 +1,211 @@
+import pytest
```

Comment:
This test file should be relocated in a test folder located in the same directory as the unit model, not in the tests folder in the surrogates directory.

Reply:
Also we need a header.